acceptance/repository-cloudant/docker.setup.js

```diff
@@ -38,7 +38,7 @@ let containerToDelete = null;
 
 async.waterfall(
   [
-    dockerStart('ibmcom/couchdb3:latest'),
+    dockerStart('ibmcom/couchdb3:preview-1575988511'),
```

Comment:
Please add a comment to explain why.

Reply:
the tag `latest` fails on creating the instance because the image got updated ([docker](https://hub.docker.com/r/ibmcom/couchdb3/tags)). Using the stable one here to pass tests.

Reply:
I meant a comment in the code.

Reply:
Ah I see :see_no_evil: